group_vars/iscsi-gws.yml.sample

```diff
@@ -58,3 +58,6 @@ dummy:
 
 #client_connections: {}
 
+# Wether or not to generate secure certificate to gateway nodes
```

Comment:
Wether -> Whether.

Do you want to clarify "gateway nodes" is "iscsi gateway nodes"? (not rados gateway)

Reply:
Done, thanks!